tests/Benchmarks/System.Binary.Base64/Base64Encode.cs

```diff
@@ -0,0 +1,43 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Attributes.Jobs;
+using BenchmarkDotNet.Engines;
+using System;
+using System.Binary.Base64Experimental;
+using System.Buffers;
+
+namespace Benchmarks.System.Binary.Base64.Benchmarks
+{
+    public class Base64Encode
+    {
+        [Params(10, 100, 1000, 1000 * 1000)]
+        public int NumberOfBytes;
+        private static readonly StandardFormat format = new StandardFormat('M');
+
+        static byte[] source;
```

Comment:
private fields should start with `_`
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md#c-coding-style

Also, please be explicit with the access modifier.